README.md

```diff
@@ -7,6 +7,118 @@ height="42" />
 
 Runtime for Newfold WP modules and plugins
 
+## Module Responsibilities
+
+* Container is loaded from the brand plugin in Runtime class and it returns a `NewfoldRuntime` object. 
+* This object can be imported in any React components as specified in `Usage` section below. 
+* Primary respnsibility of `wp-module-runtime` is to return `NewfoldRuntime` object which provides WordPress site meta data to React components of any newfold-labs modules in below format.
+
+
+```
+  {
+    "site": {
+        "url": "Your wordpress site url",
+        "title": "Your wordpress site name"
+    },
+    "adminUrl": "Your wordpress site wp-admin url",
+    "base_url": "Your wordpress site index.php url",
+    "homeUrl": "Your wordpress site url",
+    "capabilities": {
+        "canAccessAI": boolean,
+        "canAccessGlobalCTB": boolean,
+        "canAccessHelpCenter": boolean,
+        "hasEcomdash": boolean,
+        "hasYithExtended": boolean,
+        "isEcommerce": boolean,
+        "isJarvis": boolean
+    },
+    "sdk": {
```

Comment:
We should strive to remove the `sdk` property.

Reply:
@wpscholar 
To resolve https://github.com/newfold-labs/wp-module-runtime/issues/5  `$sdk` property was  removed and its contents were added at the top level using `split` operator. Which resulted in fatal error. Changes were rolledback in https://github.com/newfold-labs/wp-module-runtime/issues/10. 

Please advice, on better way to do this change. Thanks!

Reply:
Please review [JIRA](https://jira.newfold.com/browse/PRESS0-1193)

Also created `sub-tasks` under above parent ticket, for repository wise changes.